### ISSUE TITLE
Style links

### ISSUE THREAD

**Issue by eevajonnapanula** [turkupy/turkupy.github.io#70]
**Is your feature request related to a problem? Please describe.**
Currently the link styles on the page do not pass the contrast color ratio check. We also should not communicate links only with color.

**Describe the solution you'd like**
- Style the links to use bold font weight.
- Add `text-transform: underline` to the link without a state (such as hover etc.)
- Add `text-transform: none` to the link on hover and focus states

Note: This should be done in the `GlobalStyles` in the `Layout`-component

**Comment by Samyaryal** [comment]
I can take this one.

**Comment by lgrigoryeva1** [comment]
@Samyaryal there you go! :)

**Comment by sjarva** [comment]
The correct CSS property for the underlining is `text-decoration`, not `text-transform` 😃 🤗

**Comment by eevajonnapanula** [comment]
Oh yeah, thanks for noticing! I'll update the text 😅

**Comment by eevajonnapanula** [comment]
Hey @Samyaryal, how are things going with this issue? Is there anything we could do to help you? 😊

**Comment by Samyaryal** [comment]
@eevajonnapanula  Hi, Yeah maybe you could help me with that issue. :)

**Comment by eevajonnapanula** [comment]
Alright! 😊  How far have you gotten? Do you have the repo set up on your computer? Or if you have a question in mind, just go ahead, ask 😊
